### PR TITLE
Bump submodule

### PR DESCRIPTION
**Pull request by kitemongerer**

### CODE REVIEW

File: pipeline/bump-submodule/task [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+pushd submodule-repo
+  target_sha="$(git rev-parse HEAD)"
+  submodule_repo_dir=${PWD}
+popd
+
+pushd "release-repo/$SUBMODULE_PATH"
+  git fetch ${submodule_repo_dir}
+  git checkout "$target_sha"
+  submodule_name=$(basename `git rev-parse --show-toplevel`)
```
Comment (thepeterstone) [inline pipeline/bump-submodule/task]:
```suggestion
  submodule_name=$(basename $(git rev-parse --show-toplevel))
```

File: pipeline/bump-submodule/task [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+popd
+
+pushd release-repo
+  if [[ "$(git status --porcelain)" = "" ]]; then
```
Comment (thepeterstone) [inline pipeline/bump-submodule/task:19]: I'm not familiar with this - is it better than 
```bash
if git diff-index --quiet HEAD --; then
    echo "No changes to commit"
fi
```
?

**Review by thepeterstone — APPROVED** [review]
A couple small questions, but overall looks good to me!

File: pipeline/bump-submodule/task (hunk 2 above)
Comment (kitemongerer) [reply pipeline/bump-submodule/task:19]: I think that `git status --porcelain` is more readable than the other. Seeing if git status is empty is pretty clear but having to understand the exit code of diff-index is a little less obvious

**Pull request merged**